### PR TITLE
Added a slot to the Footer for a horisontal navigation.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**
Organisms___Footer_-_Footer_⋅_Storybook

### CODE REVIEW

File: components/03-organisms/navigation/navigation.scss [hunk 1]
```diff
@@ -69,7 +69,6 @@
         }
 
         .ct-menu__item {
-          margin-bottom: ct-spacing(2);
```
Comment (AlexSkrypnyk) [inline components/03-organisms/navigation/navigation.scss:72]: this existed here for no reason - I could not find any use cases were this is needed. The `inline` navigation renders within an `Item list` utility component. this component already takes care of any required spacing by using `gap`.

**Comment by richardgaunt** [comment]
@AlexSkrypnyk  From a technical perspective I am happy for this to come into CivicTheme. But I want confirmation from @alanrako on the UX nature first.

**Comment by AlexSkrypnyk** [comment]
@richardgaunt @fionamorrison23

It’s been 2 months since this PR was created, and we missed the opportunity to include it in the 1.9 release.

This feature request, driven by feedback from 2 external clients and several of Salsa Digital’s clients, proposes adding an invisible slot to extend the footer. Importantly, it doesn’t introduce any changes to the design system itself—just a straightforward enhancement to improve flexibility.

What was/is the blocker to get this merged?

**Comment by richardgaunt** [comment]
@AlexSkrypnyk Thank you Alex  for your contribution. 

I have merged this in, there is this change and a further planned enhancement to the footer but not reason why we can't do this first and then add the next.

**Pull request merged**